### PR TITLE
Update NUMA Detection, Install OpenBLAS when required

### PR DESCRIPTION
**Pull request by kdvalin**
This PR updates NUMA detection to use `detect_numa` and install openblas when required.

### CODE REVIEW

File: auto_hpl/build_run_hpl.sh [hunk 1]
```diff
@@ -581,30 +580,39 @@ run_hpl()
...
-			blaslib="${vendor}_openblas"
-		fi
-	elif [[ "$arch" == "aarch64" ]]; then
-		# Use OpenBLAS-openmp
-		echo "Using system OpenBLAS-OpenMP"
-		blaslib="aarch64_openblas"
-	fi
-	build_hpl 
-	run_hpl
+  size_platform
```
Comment (dvalinrh) [inline auto_hpl/build_run_hpl.sh:583]: Please follow indentations with tabs

**Review by dvalinrh — CHANGES_REQUESTED** [review]
Indents need to be tabs not spaces.

File: auto_hpl/build_run_hpl.sh (hunk 1 above)
Comment (kdvalin) [reply auto_hpl/build_run_hpl.sh:583]: Must've been working off an old commit, will fix

File: auto_hpl/build_run_hpl.sh [hunk 2]
```diff
@@ -691,10 +699,10 @@ if [ ${info} == "amzn2" ]; then
 	cp /usr/lib64/lib/libopenblas.so /usr/lib64/libopenblas.so
 	cp /usr/lib64/lib/libopenblas.so /usr/lib64/libopenblas.so.0
 fi
-info=`uname -a | cut -d' ' -f 4 | cut -d'-' -f2`
+
 ubuntu=0
-if [ ${info} == "Ubuntu" ]; then
-	ubuntu=1
+if [ "`test_tools/detect_os`" == "ubuntu" ]; then
+  ubuntu=1
```
Comment (dvalinrh) [inline auto_hpl/build_run_hpl.sh:705]: Need to be indented a tab.

**Review by dvalinrh — CHANGES_REQUESTED** [review]

**Pull request merged**